components/custom_status/custom_status.scss

```diff
@@ -153,6 +157,9 @@
         align-items: center;
 
         .status {
+            background: var(--sidebar-header-bg);
+            color: var(--sidebar-header-text-color);
```

Comment:
`global_header.scss` adds GH specific variables for background a text using the appropriate existing variables as confirmed with Design:

```css
    --global-header-background: var(--sidebar-teambar-bg);
    --global-header-text: var(--sidebar-header-text-color);
    --global-header-background-rgb: var(--sidebar-teambar-bg-rgb);
    --global-header-text-rgb: var(--sidebar-header-text-color-rgb);
```

So I'm wondering if maybe we should use those variables instead to be consistent and if we should move this whole chunk of GH specific CSS to `global_header.scss` to keep all styling specifically for when the GH is enabled together?